src/ray/common/memory_monitor.h

```diff
@@ -100,7 +100,9 @@ class MemoryMonitor {
 
   /// \param stat_path file path to the memory.stat file.
   /// \param usage_path file path to the memory.current file
-  /// \return the used memory for cgroup v2.
+  /// \return the used memory for cgroup v2. Note that the result may be negative,
+  /// which means that there is more inactive file memory than currently used memory. The
```

Comment:
Doesn't used include inactive, and hence should be greater?

If used is less sounds like there is some underlying issue?

I think its okay to just say it may return a negative value